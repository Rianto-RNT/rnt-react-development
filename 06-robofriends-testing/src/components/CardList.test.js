import React from 'react';
import { shallow } from 'enzyme';
import CardList from './CardList';
const filteredRobots = [{
    id: 1,
    name: 'Rianto RNT',
    username: 'rian',
    email: 'rian@rnt.io'
  }]

it('renders without crashing', () => {
  expect(shallow(<CardList robots={filteredRobots}/>)).toMatchSnapshot();
});