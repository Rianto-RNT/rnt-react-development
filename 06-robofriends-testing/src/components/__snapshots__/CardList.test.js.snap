// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<div>
  <Card
    email="rian@rnt.io"
    id={1}
    key="0"
    name="Rianto RNT"
  />
</div>
`;
